filebeat/module/elasticsearch/audit/ingest/pipeline-json.json

```diff
@@ -0,0 +1,197 @@
+{
+    "description": "Pipeline for parsing elasticsearch audit logs in JSON format",
+    "processors": [
+        {
+            "json": {
+                "field": "message",
+                "target_field": "elasticsearch.audit"
+            }
+        },
+        {
+            "dot_expander": {
+                "field": "event.action",
+                "path": "elasticsearch.audit"
+            }
+        },
+        {
+            "rename": {
+                "field": "elasticsearch.audit.event.action",
+                "target_field": "elasticsearch.audit.event_type"
+            }
+        },
+        {
+            "dot_expander": {
```

Comment:
This is only need to make the event look nicer?

Reply:
No, actually (and unfortunately IMO), it is required for the next processor (`rename`) to work. If I remove this `dot_expander` processor entry, I will get an error like so from ES when it tries to execute the `rename` processor:

```
field [elasticsearch.audit.event.type] doesn't exist
```

Reply:
Perhaps we should file an enhancement request around this with ES

Reply:
https://github.com/elastic/elasticsearch/issues/37507